examples/asr/jasper.py

```diff
@@ -43,18 +44,18 @@ def parse_args():
     parser.add_argument("--warmup_steps", default=0, type=int)
 
     args = parser.parse_args()
-    if args.max_steps is not None:
-        raise ValueError("Jasper uses num_epochs instead of max_steps")
 
+    if args.max_steps is not None and args.num_epochs is not None:
+        raise ValueError("Either max_steps or num_epochs should be provided.")
     return args
 
 
-def construct_name(name, lr, batch_size, num_epochs, wd, optimizer,
+def construct_name(name, lr, batch_size, max_steps, wd, optimizer,
                    iter_per_step):
-    return ("{0}-lr_{1}-bs_{2}-e_{3}-wd_{4}-opt_{5}-ips_{6}".format(
+    return ("{0}-lr_{1}-bs_{2}-s_{3}-wd_{4}-opt_{5}-ips_{6}".format(
         name, lr,
         batch_size,
-        num_epochs,
+        max_steps,
```

Comment:
This will output "s_None" if a user uses num_epochs instead.

Reply:
fixed